### ISSUE TITLE
illuminate/support required version sould be bumped

### ISSUE THREAD

**Issue by Dartui** [BenSampo/laravel-enum#34]
Hi,

first of all, thanks for creating this great package. It is very useful, so keep maintaining it! :slightly_smiling_face: 

In composer.json there are two requirements: `php` in version 7.0 or higher and `illuminate\support`  in version 5.1 or higher. This is a bit confusing, because `illuminate/support` in 5.1 requires php in at least 5.5.9 version. Also if you try using this package with Laravel 5.4, which requires php in version 5.6.4, you cannot use validation rules - `Illuminate\Contracts\Validation\Rule` was introduces in Laravel 5.5.

To sum up: you can install this package on server with php in version 7.0 or higher and with Laravel in versions between 5.1 and 5.4, but in this configuration you cannot use validation rules.

What do you think about bumping version of `illuminate/support` to 5.5?
